javascript/chronometer.js

```diff
@@ -1,26 +1,33 @@
 class Chronometer {
-  constructor() {
-    // ... your code goes here
-  }
-  startClick(callback) {
-    // ... your code goes here
-  }
+  constructor() {  //should not receive any arguments
+    this.currentTime = 0; //stablish the currentTime property
+    this.intervalId = 0; //Stablish the intervalID property
+  }
+  startClick(callback) { //if every second currentTime increment by one, every 3 seconds it will increment by 3
+    let intervalId = setInterval(() => {
+			this.currentTime++;
+		}, 1000);
+	}
+   
   getMinutes() {
-    // ... your code goes here
-  }
+    return Math.floor(this.currentTime / 60);  //i used math.floor for return a number without decimals.
+    }
+
   getSeconds() {
-    // ... your code goes here
-  }
+    return Math.floor(this.currentTime % 60);  //i used math.floor for return a number without decimals.
+    }
+
   twoDigitsNumber() {
-    // ... your code goes here
+    return ("0" + this.currentTime).slice(-2);
   }
+
   stopClick() {
-    // ... your code goes here
+    return clearInterval (this.currentTime);
```

Comment:
this clearInterval won't work, what you want to pass inside the parenthesis is the intervalId, not the counter you have. The intervalId you have it saved in a variable in L7. If you save it in the property of the class, the you can easily use it here. (this.intervalId)